client/my-sites/upgrades/trials/start-trial/free-trial-confirmation-box.js

```diff
@@ -35,6 +35,15 @@ const FreeTrialConfirmationBox = React.createClass( {
 				<TermsOfService />
 				<div className="payment-box-actions">
 					<PayButton
+						beforeSubmitText={
+							this.translate( 'Start %(days)s Day Free Trial', {
```

Comment:
I have just moved this part from `checkout/pay-button.jsx` so I didn't take any decision on this. But I personally think it makes the translation more generic. Any reason not to use arguments?